### ISSUE TITLE
nxt clone 

### ISSUE THREAD

**Issue by ben-haim** [fimkrypto/mofowallet#14]
Hi there,
I would like whether i could use this as nxt wallet ? it seem i could swap fim ( id ) is that make sense ?
Best Regards,
ychaim

**Comment by fitcoin** [comment]
Yes it's fully possible. NXT has been supported in earlier versions.
